src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -1970,7 +1972,7 @@ void CheckClosed()
                 throw new InvalidOperationException("The reader is closed");
         }
 
-        #endregion
+#endregion
```

Comment:
As elsewhere, our style is currently to indent region directives.

Reply:
Fixed.